WordPress/src/main/java/org/wordpress/android/models/Person.java

```diff
@@ -1,33 +1,62 @@
 package org.wordpress.android.models;
 
+import org.json.JSONObject;
+
 public class Person {
-    public long personID;
+    private long personID;
+    private int localTableBlogId;
 
     private String username;
     private String firstName;
     private String lastName;
     private String displayName;
-    private String imageUrl;
+    private String avatarUrl;
     private Role role;
 
     public Person(long personID,
+                  int localTableBlogId,
                   String username,
                   String firstName,
                   String lastName,
                   String displayName,
-                  String imageUrl,
+                  String avatarUrl,
                   Role role) {
         this.personID = personID;
+        this.localTableBlogId = localTableBlogId;
         this.username = username;
         this.firstName = firstName;
         this.lastName = lastName;
         this.displayName = displayName;
-        this.imageUrl = imageUrl;
+        this.avatarUrl = avatarUrl;
         this.role = role;
     }
 
+    public static Person fromJSON(JSONObject json, int localTableBlogId) {
+        if (json == null) {
+            return null;
+        }
+
+        long personID = Long.parseLong(json.optString("ID"));
```

Comment:
Person id seems to be read as an optional field from the json. Is it really? If yes, then `parseLong` will fail with an Unchecked exception force-closing the app. If person id is indeed optional then we need to handle this.

OTOH, if the person id is not optional in the json/api then the more expressive `getString()` better be used. That one will also force us to handle the potential error in the json.

All in all, we need to handle any potential error here since `personID` is part of the DB table's key and better be trustworthy. Cheers!